packages/valory/agents/impact_evaluator/aea-config.yaml

```diff
@@ -90,7 +90,7 @@ logging_config:
 skill_exception_policy: stop_and_exit
 dependencies:
   open-aea-ledger-ethereum:
-    version: ==1.53.0
+    version: ==1.58.0
```

Comment:
@Adamantios both agent and service yaml are missing some gas configuration in order to properly work

Reply:
You will only need to override the value if you don't want to use dynamic gas pricing or if you want to use a testnet.